### PR TITLE
Brings NTPR to the nice discord bot

### PR DESCRIPTION
**Pull request by Poojawa**
:cl: Poojawa
server: Discord bot will now report when a round ends and give the vague news ticker that a sister-station would have received for the game mode and result of the round.
/:cl:

Why is it only new game starting and meme commands that work?

:v

### CODE REVIEW

File: code/__HELPERS/roundend.dm [hunk 1]
```diff
@@ -199,6 +199,10 @@
 	if(length(CONFIG_GET(keyed_list/cross_server)))
 		send_news_report()
 
+	//tell the nice people on discord what went on before the salt cannon happens.
+	world.TgsTargetedChatBroadcast("The current round has ended. Please standby for your shift interlude Nanotrasen News Network's report!", FALSE)
+	world.TgsTargetedChatBroadcast(send_news_report(),FALSE)
```
Comment (deathride58) [inline code/__HELPERS/roundend.dm:204]: TGS doesn't automatically account for rate limiting when outputting to Discord, and Discord's rate limit is pretty strict last I checked. Might be a good idea to use only one targetedchatbroadcast here

**Review by deathride58 — CHANGES_REQUESTED** [review]
Goodsune,

**Comment by Thalpy** [comment]
It's testament to your character that you can make such a pr without filling all output discord messages with blobcats.

File: code/__HELPERS/roundend.dm (hunk 1 above)
Comment (Poojawa) [reply code/__HELPERS/roundend.dm:204]: Maybe. It'd make the message pretty messy tho.

**Comment by Poojawa** [comment]
> It's testament to your character that you can make such a pr without filling all output discord messages with blobcats.

With great power comes great responsibilities.

File: code/__HELPERS/roundend.dm (hunk 1 above)
Comment (deathride58) [reply code/__HELPERS/roundend.dm:204]: p sure you can pass a linebreak through it somehow

Comment (Poojawa) [reply code/__HELPERS/roundend.dm:204]: I'll see how it does first, then just make it a blob of msg += s

**Review by deathride58 — APPROVED** [review]

**Pull request merged**